### PR TITLE
Add Lock iOS: Sign Up

### PR DESCRIPTION
**Pull request by lobo**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/libraries/lock-ios/index.md [hunk 1]
```diff
@@ -180,6 +180,128 @@ controller.onUserDismissBlock = { () in
 self.presentViewController(controller, animated: true, completion: nil)
 ```
 
+### Sign Up
+
+There are different approaches to implement the "Sign Up" functionality.
+
+1. You can just add or hide "Sign Up" button in Lock UI. For this you need to setup property `disableSignUp` which hides the Sign Up button. The default value for this attribute is `NO`.
```
Comment (chenkie) [inline articles/libraries/lock-ios/index.md]: add or hide "Sign Up" => add or hide the "Sign Up"

File: articles/libraries/lock-ios/index.md [hunk 2]
```diff
@@ -180,6 +180,128 @@ controller.onUserDismissBlock = { () in
...
+controller.onAuthenticationBlock = { (profile, token) in
+  // Do something with token & profile. e.g.: save them.
+  // And dismiss the ViewController
+}
+
+self.presentViewController(controller, animated: true, completion: nil)
+```
+
+
+3. A third approach would be to add your own Sign Up View Controller which should implement the sign up logic with one of the `A0APIClient` method. Let's consider this, for example:
```
Comment (chenkie) [inline articles/libraries/lock-ios/index.md]: which should => which will

**Comment by lobo** [comment]
@chenkie ready 👍

**Pull request merged**